### DISCUSSION TITLE
Android status bar always white regardless of light/dark mode

### DISCUSSION THREAD

**Discussion by Dukestep** [aeharding/voyager#444] [Q&A] [answered]
I've noticed that the status bar is not themed when dark mode is active on an Android device.

<img src="https://github.com/aeharding/voyager/assets/2139901/d87fe337-4a7c-4ab3-9931-a8aec9bc41cf" width="33%" height="33%">

Would it be possible to make it black or is this an Ionic limitation (not that familiar with the framework!) ? This behaviour was spotted  on a Pixel 6 Pro running Android 14 Beta 4.

**Comment by mdecourcy** [comment]
Seems like this might be an issue with your browser or android 14, as I'm on a pixel 6 running graphene OS and mine is fine 

![Screenshot_20230714-153223.png](https://github.com/aeharding/voyager/assets/49794076/e5e4cf0c-0c71-4f66-a031-04c8b1a98115)

**Comment by Dukestep** [comment, marked as answer]
@mdecourcy thanks for narrowing it down. I installed it via Chrome. Suspect it's an issue with Android 14 beta... There's nothing too snazzy or different about my setup other than that.

I'll let others who may be on the last Android beta chime in. Might be fixed upstream eventually!
